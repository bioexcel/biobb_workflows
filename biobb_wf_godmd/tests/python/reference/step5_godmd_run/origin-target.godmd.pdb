MODEL           0
ATOM      1 CA   MET A1        -11.468 -22.766 -12.854
ATOM      2 CA   ARG A2         -8.607 -21.023 -11.035
ATOM      3 CA   ILE A3         -6.076 -19.085 -13.125
ATOM      4 CA   ILE A4         -2.852 -17.131 -12.373
ATOM      5 CA   LEU A5         -1.756 -14.485 -14.950
ATOM      6 CA   LEU A6          2.050 -13.927 -14.881
ATOM      7 CA   GLY A7          3.959 -11.257 -16.896
ATOM      8 CA   ALA A8          6.101  -8.062 -16.793
ATOM      9 CA   PRO A9          4.456  -4.616 -16.164
ATOM     10 CA   GLY A10         2.163  -3.717 -19.129
ATOM     11 CA   ALA A11         2.231  -7.214 -20.741
ATOM     12 CA   GLY A12        -1.586  -7.055 -21.172
ATOM     13 CA   LYS A13        -2.655  -9.119 -18.119
ATOM     14 CA   GLY A14        -5.436  -6.787 -16.944
ATOM     15 CA   THR A15        -6.768  -6.351 -20.561
ATOM     16 CA   GLN A16        -7.081 -10.163 -20.982
ATOM     17 CA   ALA A17        -8.410 -10.671 -17.461
ATOM     18 CA   GLN A18       -11.586  -8.642 -18.458
ATOM     19 CA   PHE A19       -12.187 -11.115 -21.407
ATOM     20 CA   ILE A20       -11.647 -14.255 -19.319
ATOM     21 CA   MET A21       -13.932 -12.887 -16.681
ATOM     22 CA   GLU A22       -16.760 -12.033 -19.071
ATOM     23 CA   LYS A23       -16.417 -15.245 -21.085
ATOM     24 CA   TYR A24       -15.874 -17.739 -18.260
ATOM     25 CA   GLY A25       -17.821 -15.999 -15.489
ATOM     26 CA   ILE A26       -14.924 -16.172 -12.953
ATOM     27 CA   PRO A27       -14.076 -13.087 -10.854
ATOM     28 CA   GLN A28       -10.926 -11.216 -11.476
ATOM     29 CA   ILE A29        -9.011 -10.619  -8.146
ATOM     30 CA   SER A30        -6.500  -7.859  -9.073
ATOM     31 CA   THR A31        -4.251  -6.774  -6.318
ATOM     32 CA   GLY A32        -3.290  -3.505  -8.090
ATOM     33 CA   ASP A33        -7.015  -2.665  -8.317
ATOM     34 CA   MET A34        -7.676  -3.627  -4.698
ATOM     35 CA   LEU A35        -4.699  -1.578  -3.452
ATOM     36 CA   ARG A36        -5.488   1.673  -5.495
ATOM     37 CA   ALA A37        -9.046   1.318  -4.186
ATOM     38 CA   ALA A38        -7.835   0.925  -0.551
ATOM     39 CA   VAL A39        -5.543   3.948  -0.938
ATOM     40 CA   LYS A40        -8.442   5.938  -2.370
ATOM     41 CA   SER A41       -10.933   4.835   0.398
ATOM     42 CA   GLY A42        -8.384   5.179   3.253
ATOM     43 CA   SER A43        -9.300   1.491   4.068
ATOM     44 CA   GLU A44        -7.187   0.291   7.021
ATOM     45 CA   LEU A45        -6.125  -2.831   5.070
ATOM     46 CA   GLY A46        -4.213  -1.382   2.182
ATOM     47 CA   LYS A47        -4.121   2.456   2.525
ATOM     48 CA   GLN A48        -0.466   1.937   3.568
ATOM     49 CA   ALA A49         0.493   1.105  -0.056
ATOM     50 CA   LYS A50         0.192   4.705  -1.212
ATOM     51 CA   ASP A51         3.835   5.687  -1.121
ATOM     52 CA   ILE A52         5.306   2.345  -2.083
ATOM     53 CA   MET A53         3.230   2.187  -5.327
ATOM     54 CA   ASP A54         4.203   5.793  -6.065
ATOM     55 CA   ALA A55         7.837   4.877  -5.806
ATOM     56 CA   GLY A56         7.341   1.833  -8.187
ATOM     57 CA   LYS A57         8.084  -0.648  -5.499
ATOM     58 CA   LEU A58         6.430  -3.894  -4.405
ATOM     59 CA   VAL A59         4.078  -3.625  -1.340
ATOM     60 CA   THR A60         5.039  -6.037   1.580
ATOM     61 CA   ASP A61         4.027  -9.625   1.040
ATOM     62 CA   GLU A62         2.371  -9.794   4.438
ATOM     63 CA   LEU A63        -0.058  -6.978   3.439
ATOM     64 CA   VAL A64        -0.983  -8.241   0.024
ATOM     65 CA   ILE A65        -1.449 -11.888   1.190
ATOM     66 CA   ALA A66        -3.938 -10.554   3.826
ATOM     67 CA   LEU A67        -6.030  -8.750   1.127
ATOM     68 CA   VAL A68        -6.130 -11.819  -1.149
ATOM     69 CA   LYS A69        -7.128 -14.253   1.590
ATOM     70 CA   GLU A70        -9.959 -11.940   2.506
ATOM     71 CA   ARG A71       -11.179 -11.693  -1.037
ATOM     72 CA   ILE A72       -10.979 -15.345  -1.839
ATOM     73 CA   ALA A73       -13.265 -15.842   1.219
ATOM     74 CA   GLN A74       -16.177 -13.898  -0.416
ATOM     75 CA   GLU A75       -18.915 -16.108  -1.813
ATOM     76 CA   ASP A76       -18.021 -15.329  -5.479
ATOM     77 CA   CYS A77       -14.689 -17.168  -5.180
ATOM     78 CA   ARG A78       -16.063 -20.635  -4.174
ATOM     79 CA   ASN A79       -16.326 -21.485  -7.878
ATOM     80 CA   GLY A80       -13.033 -20.134  -9.199
ATOM     81 CA   PHE A81       -11.300 -16.912  -9.649
ATOM     82 CA   LEU A82        -8.626 -15.351 -11.791
ATOM     83 CA   LEU A83        -5.487 -13.938  -9.935
ATOM     84 CA   ASP A84        -4.163 -10.945 -11.779
ATOM     85 CA   GLY A85        -1.061  -9.479 -10.181
ATOM     86 CA   PHE A86        -0.707 -12.048  -7.438
ATOM     87 CA   PRO A87         1.546 -13.929  -6.597
ATOM     88 CA   ARG A88         4.184 -11.395  -7.312
ATOM     89 CA   THR A89         6.980 -12.941  -5.260
ATOM     90 CA   ILE A90         8.051 -16.454  -4.143
ATOM     91 CA   PRO A91         6.773 -15.801  -0.482
ATOM     92 CA   GLN A92         3.333 -14.948  -1.936
ATOM     93 CA   ALA A93         3.436 -18.182  -4.050
ATOM     94 CA   ASP A94         4.466 -20.243  -1.012
ATOM     95 CA   ALA A95         1.665 -18.676   1.047
ATOM     96 CA   MET A96        -0.869 -19.880  -1.565
ATOM     97 CA   LYS A97         0.737 -23.346  -1.335
ATOM     98 CA   GLU A98         0.558 -23.457   2.514
ATOM     99 CA   ALA A99        -3.075 -22.337   2.342
ATOM    100 CA   GLY A100       -4.425 -25.045  -0.064
ATOM    101 CA   ILE A101       -4.957 -22.660  -3.031
ATOM    102 CA   ASN A102       -3.951 -24.676  -6.033
ATOM    103 CA   VAL A103       -4.746 -23.359  -9.578
ATOM    104 CA   ASP A104       -5.644 -24.912 -12.818
ATOM    105 CA   TYR A105       -3.648 -22.757 -15.225
ATOM    106 CA   VAL A106       -0.756 -20.387 -15.017
ATOM    107 CA   LEU A107       -0.517 -18.164 -18.152
ATOM    108 CA   GLU A108        2.640 -16.247 -18.847
ATOM    109 CA   PHE A109        2.272 -13.180 -21.036
ATOM    110 CA   ASP A110        5.698 -12.616 -22.616
ATOM    111 CA   VAL A111        6.605  -9.184 -23.885
ATOM    112 CA   PRO A112       10.113  -7.688 -24.539
ATOM    113 CA   ASP A113       11.253  -5.009 -22.013
ATOM    114 CA   GLU A114       11.684  -2.441 -24.790
ATOM    115 CA   LEU A115        7.935  -2.254 -25.557
ATOM    116 CA   ILE A116        6.801  -1.538 -21.994
ATOM    117 CA   VAL A117        7.368   2.147 -21.533
ATOM    118 CA   ASP A118        5.179   3.131 -24.633
ATOM    119 CA   ARG A119        2.417   0.727 -23.706
ATOM    120 CA   ILE A120        2.223   2.293 -20.342
ATOM    121 CA   VAL A121        2.297   5.991 -21.414
ATOM    122 CA   GLY A122       -0.779   5.516 -23.599
ATOM    123 CA   ARG A123       -2.638   3.664 -20.794
ATOM    124 CA   ARG A124       -5.922   5.207 -19.516
ATOM    125 CA   VAL A125       -8.160   3.565 -16.957
ATOM    126 CA   HIS A126      -11.657   3.680 -15.608
ATOM    127 CA   ALA A127      -10.687   3.484 -11.915
ATOM    128 CA   PRO A128      -14.002   2.399 -10.369
ATOM    129 CA   SER A129      -14.058  -0.715 -12.491
ATOM    130 CA   GLY A130      -10.440  -1.237 -13.234
ATOM    131 CA   ARG A131      -11.228  -1.374 -17.026
ATOM    132 CA   VAL A132       -8.121  -0.393 -19.032
ATOM    133 CA   TYR A133       -7.651   1.209 -22.449
ATOM    134 CA   HIS A134       -4.739   2.441 -24.616
ATOM    135 CA   VAL A135       -5.032   5.617 -26.676
ATOM    136 CA   LYS A136       -3.477   3.848 -29.710
ATOM    137 CA   PHE A137       -3.633   0.118 -29.214
ATOM    138 CA   ASN A138       -7.031  -0.389 -27.744
ATOM    139 CA   PRO A139       -8.868   2.932 -27.570
ATOM    140 CA   PRO A140      -12.359   3.557 -25.989
ATOM    141 CA   LYS A141      -15.349   3.848 -28.549
ATOM    142 CA   VAL A142      -15.647   7.486 -27.528
ATOM    143 CA   GLU A143      -12.317   9.143 -26.789
ATOM    144 CA   GLY A144      -11.658   9.710 -23.099
ATOM    145 CA   LYS A145      -14.748   7.915 -21.795
ATOM    146 CA   ASP A146      -15.319   4.495 -20.449
ATOM    147 CA   ASP A147      -17.470   2.336 -22.844
ATOM    148 CA   VAL A148      -19.842   1.095 -20.173
ATOM    149 CA   THR A149      -20.451   4.045 -17.939
ATOM    150 CA   GLY A150      -19.463   6.990 -20.128
ATOM    151 CA   GLU A151      -17.239   8.202 -17.312
ATOM    152 CA   GLU A152      -14.042  10.078 -17.957
ATOM    153 CA   LEU A153      -10.889   7.941 -18.144
ATOM    154 CA   THR A154       -7.796   8.963 -16.083
ATOM    155 CA   THR A155       -4.088   8.106 -16.052
ATOM    156 CA   ARG A156       -2.503   6.212 -13.187
ATOM    157 CA   LYS A157       -0.173   8.201 -10.982
ATOM    158 CA   ASP A158        2.574   5.524 -11.228
ATOM    159 CA   ASP A159        2.461   5.389 -15.064
ATOM    160 CA   GLN A160        5.348   7.704 -15.810
ATOM    161 CA   GLU A161        8.530   6.569 -17.506
ATOM    162 CA   GLU A162       10.741   6.509 -14.383
ATOM    163 CA   THR A163        8.304   4.661 -12.086
ATOM    164 CA   VAL A164        7.767   2.066 -14.898
ATOM    165 CA   ARG A165       11.591   1.574 -15.089
ATOM    166 CA   LYS A166       11.717   0.971 -11.401
ATOM    167 CA   ARG A167        8.819  -1.460 -11.663
ATOM    168 CA   LEU A168       10.668  -3.417 -14.434
ATOM    169 CA   VAL A169       13.790  -3.924 -12.261
ATOM    170 CA   GLU A170       11.643  -5.145  -9.281
ATOM    171 CA   TYR A171        9.829  -7.452 -11.716
ATOM    172 CA   HIS A172       13.128  -9.019 -12.892
ATOM    173 CA   GLN A173       14.479  -9.343  -9.391
ATOM    174 CA   MET A174       11.409 -10.655  -7.527
ATOM    175 CA   THR A175        8.675 -11.759  -9.871
ATOM    176 CA   ALA A176       10.460 -13.239 -12.851
ATOM    177 CA   PRO A177       11.503 -16.211 -10.611
ATOM    178 CA   LEU A178        7.848 -17.302 -10.509
ATOM    179 CA   ILE A179        8.234 -18.503 -14.166
ATOM    180 CA   GLY A180       10.630 -21.262 -12.972
ATOM    181 CA   TYR A181        8.559 -21.850  -9.785
ATOM    182 CA   TYR A182        5.318 -22.659 -11.644
ATOM    183 CA   SER A183        7.062 -24.335 -14.544
ATOM    184 CA   LYS A184        8.322 -26.776 -11.855
ATOM    185 CA   GLU A185        4.741 -27.002 -10.555
ATOM    186 CA   ALA A186        3.526 -27.938 -14.089
ATOM    187 CA   GLU A187        6.245 -30.599 -14.433
ATOM    188 CA   ALA A188        5.015 -32.210 -11.256
ATOM    189 CA   GLY A189        1.406 -32.395 -12.528
ATOM    190 CA   ASN A190        0.091 -29.713 -10.034
ATOM    191 CA   THR A191       -1.075 -27.123 -12.681
ATOM    192 CA   LYS A192       -0.989 -26.486 -16.485
ATOM    193 CA   TYR A193        1.537 -23.690 -17.623
ATOM    194 CA   ALA A194        0.979 -21.906 -21.032
ATOM    195 CA   LYS A195        3.114 -19.099 -22.467
ATOM    196 CA   VAL A196        1.382 -16.558 -24.859
ATOM    197 CA   ASP A197        2.937 -13.980 -27.131
ATOM    198 CA   GLY A198        1.890 -10.697 -25.564
ATOM    199 CA   THR A199        3.195  -8.646 -28.634
ATOM    200 CA   LYS A200        0.264  -9.833 -30.806
ATOM    201 CA   PRO A201       -2.660  -7.522 -31.204
ATOM    202 CA   VAL A202       -5.188  -7.581 -28.292
ATOM    203 CA   ALA A203       -7.778  -9.646 -30.305
ATOM    204 CA   GLU A204       -5.147 -12.097 -31.400
ATOM    205 CA   VAL A205       -3.894 -12.610 -27.826
ATOM    206 CA   ARG A206       -7.595 -13.045 -26.939
ATOM    207 CA   ALA A207       -7.928 -15.822 -29.551
ATOM    208 CA   ASP A208       -4.766 -17.759 -28.314
ATOM    209 CA   LEU A209       -6.245 -17.715 -24.867
ATOM    210 CA   GLU A210       -9.561 -19.240 -26.070
ATOM    211 CA   LYS A211       -7.509 -22.097 -27.589
ATOM    212 CA   ILE A212       -5.915 -22.778 -24.227
ATOM    213 CA   LEU A213       -9.109 -22.436 -22.072
ATOM    214 CA   GLY A214      -12.140 -23.702 -24.129
ENDMDL
