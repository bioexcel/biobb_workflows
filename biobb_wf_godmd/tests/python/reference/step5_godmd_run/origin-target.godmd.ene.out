Time     2.58 Epot  -153.375 Temp(K)   48.21 Dener         0.00 Salto    0 Error   0
Time     5.17 Epot  -156.062 Temp(K)   50.46 Dener         0.00 Salto    0 Error   0
Time     7.74 Epot  -153.312 Temp(K)   47.23 Dener         0.00 Salto    0 Error   0
Time    10.32 Epot  -154.062 Temp(K)   50.66 Dener         0.00 Salto    0 Error   0
Time    12.91 Epot  -155.015 Temp(K)   48.48 Dener    210567.22 Salto    1 Error   0
Time    16.53 Epot  -143.994 Temp(K)   45.63 Dener    210567.22 Salto    1 Error   0
Time    19.73 Epot  -146.995 Temp(K)   49.08 Dener    210567.22 Salto    1 Error   0
Time    22.84 Epot  -145.829 Temp(K)   47.84 Dener    210567.22 Salto    1 Error   0
Time    26.47 Epot  -149.321 Temp(K)   49.75 Dener    210567.22 Salto    1 Error   0
Time    30.19 Epot  -147.439 Temp(K)   50.57 Dener    210567.22 Salto    1 Error   0
Time    33.60 Epot  -142.941 Temp(K)   44.77 Dener    210567.22 Salto    1 Error   0
Time    37.33 Epot  -143.716 Temp(K)   46.49 Dener    341507.41 Salto    2 Error   0
Time    40.85 Epot  -144.495 Temp(K)   49.05 Dener    341507.41 Salto    2 Error   0
Time    43.74 Epot  -138.903 Temp(K)   48.70 Dener    341507.41 Salto    2 Error   0
Time    47.57 Epot  -141.942 Temp(K)   43.07 Dener    341507.41 Salto    2 Error   0
Time    50.55 Epot  -139.269 Temp(K)   48.74 Dener    341507.41 Salto    2 Error   0
Time    53.56 Epot  -137.968 Temp(K)   46.95 Dener    341507.41 Salto    2 Error   0
Time    58.12 Epot  -141.689 Temp(K)   51.16 Dener    341507.41 Salto    2 Error   0
Time    61.34 Epot  -137.495 Temp(K)   49.11 Dener    341507.41 Salto    2 Error   0
Time    64.86 Epot  -137.130 Temp(K)   46.35 Dener    341507.41 Salto    2 Error   0
Time    68.08 Epot  -135.320 Temp(K)   48.74 Dener    341507.41 Salto    2 Error   0
Time    71.80 Epot  -137.461 Temp(K)   51.00 Dener    341507.41 Salto    2 Error   0
Time    75.84 Epot  -137.358 Temp(K)   48.63 Dener    341507.41 Salto    2 Error   0
Time    79.27 Epot  -135.148 Temp(K)   49.32 Dener    435830.22 Salto    3 Error   0
Time    82.57 Epot  -134.720 Temp(K)   50.09 Dener    435830.22 Salto    3 Error   0
Time    86.20 Epot  -133.748 Temp(K)   46.88 Dener    435830.22 Salto    3 Error   0
Time    89.82 Epot  -133.438 Temp(K)   46.80 Dener    435830.22 Salto    3 Error   0
Time    93.81 Epot  -133.161 Temp(K)   46.13 Dener    435830.22 Salto    3 Error   0
Time    97.76 Epot  -135.796 Temp(K)   49.81 Dener    435830.22 Salto    3 Error   0
Time   101.47 Epot  -128.997 Temp(K)   47.27 Dener    522622.88 Salto    4 Error   0
Time   105.20 Epot  -128.210 Temp(K)   47.44 Dener    522622.88 Salto    4 Error   0
Time   108.62 Epot  -127.830 Temp(K)   44.96 Dener    522622.88 Salto    4 Error   0
Time   112.56 Epot  -131.557 Temp(K)   46.64 Dener    522622.88 Salto    4 Error   0
Time   115.76 Epot  -128.468 Temp(K)   48.50 Dener    594812.44 Salto    5 Error   0
Time   119.99 Epot  -126.757 Temp(K)   46.89 Dener    594812.44 Salto    5 Error   0
Time   123.83 Epot  -129.386 Temp(K)   48.86 Dener    594812.44 Salto    5 Error   0
Time   128.17 Epot  -131.405 Temp(K)   47.38 Dener    594812.44 Salto    5 Error   0
Time   131.49 Epot  -130.808 Temp(K)   47.45 Dener    594812.44 Salto    5 Error   0
Time   135.31 Epot  -134.845 Temp(K)   51.99 Dener    594812.44 Salto    5 Error   0
Time   139.24 Epot  -137.303 Temp(K)   48.53 Dener    594812.44 Salto    5 Error   0
Time   142.23 Epot  -131.698 Temp(K)   48.42 Dener    594812.44 Salto    5 Error   0
Time   146.80 Epot  -131.165 Temp(K)   49.79 Dener    594812.44 Salto    5 Error   0
Time   151.54 Epot  -128.009 Temp(K)   48.18 Dener    672543.44 Salto    6 Error   0
Time   155.58 Epot  -127.425 Temp(K)   46.51 Dener    672543.44 Salto    6 Error   0
Time   159.93 Epot  -127.178 Temp(K)   47.97 Dener    672543.44 Salto    6 Error   0
Time   164.27 Epot  -127.172 Temp(K)   50.09 Dener    672543.44 Salto    6 Error   0
Time   168.71 Epot  -129.669 Temp(K)   48.31 Dener    672543.44 Salto    6 Error   0
Time   172.95 Epot  -124.607 Temp(K)   49.47 Dener    747539.25 Salto    7 Error   0
Time   176.86 Epot  -119.042 Temp(K)   45.01 Dener    747539.25 Salto    7 Error   0
Time   182.23 Epot  -124.143 Temp(K)   47.03 Dener    747539.25 Salto    7 Error   0
Time   186.28 Epot  -123.459 Temp(K)   49.90 Dener    747539.25 Salto    7 Error   0
Time   191.04 Epot  -120.326 Temp(K)   48.77 Dener    747539.25 Salto    7 Error   0
Time   195.26 Epot  -118.152 Temp(K)   47.21 Dener    803445.94 Salto    8 Error   0
Time   200.10 Epot  -120.937 Temp(K)   48.23 Dener    803445.94 Salto    8 Error   0
Time   203.71 Epot  -121.136 Temp(K)   47.63 Dener    803445.94 Salto    8 Error   0
Time   208.68 Epot  -120.559 Temp(K)   48.45 Dener    803445.94 Salto    8 Error   0
Time   212.60 Epot  -119.177 Temp(K)   49.06 Dener    803445.94 Salto    8 Error   0
Time   217.25 Epot  -119.999 Temp(K)   48.19 Dener    847407.31 Salto    9 Error   0
Time   222.53 Epot  -120.570 Temp(K)   48.14 Dener    847407.31 Salto    9 Error   0
Time   229.25 Epot  -121.560 Temp(K)   47.85 Dener    847407.31 Salto    9 Error   0
Time   234.31 Epot  -121.535 Temp(K)   46.67 Dener    847407.31 Salto    9 Error   0
Time   239.69 Epot  -122.069 Temp(K)   49.73 Dener    847407.31 Salto    9 Error   0
Time   245.17 Epot  -118.772 Temp(K)   50.77 Dener    847407.31 Salto    9 Error   0
Time   249.20 Epot  -117.255 Temp(K)   49.36 Dener    885225.06 Salto   10 Error   0
Time   255.83 Epot  -119.764 Temp(K)   49.96 Dener    885225.06 Salto   10 Error   0
Time   260.55 Epot  -121.099 Temp(K)   49.97 Dener    885225.06 Salto   10 Error   0
Time   266.98 Epot  -121.303 Temp(K)   46.18 Dener    885225.06 Salto   10 Error   0
Time   271.95 Epot  -115.788 Temp(K)   45.72 Dener    885225.06 Salto   10 Error   0
Time   277.95 Epot  -122.201 Temp(K)   48.35 Dener    885225.06 Salto   10 Error   0
Time   287.78 Epot  -125.015 Temp(K)   50.19 Dener    885225.06 Salto   10 Error   0
Time   295.02 Epot  -123.511 Temp(K)   48.17 Dener    885225.06 Salto   10 Error   0
Time   302.88 Epot  -128.119 Temp(K)   47.66 Dener    885225.06 Salto   10 Error   0
Time   308.56 Epot  -128.912 Temp(K)   48.25 Dener    885225.06 Salto   10 Error   0
Time   315.58 Epot  -131.601 Temp(K)   47.41 Dener    885225.06 Salto   10 Error   0
Time   320.85 Epot  -133.238 Temp(K)   49.78 Dener    885225.06 Salto   10 Error   0
Time   325.81 Epot  -132.542 Temp(K)   49.18 Dener    885225.06 Salto   10 Error   0
Time   330.96 Epot  -131.363 Temp(K)   46.15 Dener    885225.06 Salto   10 Error   0
Time   335.09 Epot  -136.799 Temp(K)   46.58 Dener    885225.06 Salto   10 Error   0
Time   338.92 Epot  -134.444 Temp(K)   45.28 Dener    885225.06 Salto   10 Error   0
Time   344.27 Epot  -138.425 Temp(K)   48.73 Dener    885225.06 Salto   10 Error   0
Time   351.19 Epot  -140.537 Temp(K)   47.64 Dener    885225.06 Salto   10 Error   0
Time   354.81 Epot  -143.917 Temp(K)   51.21 Dener    885225.06 Salto   10 Error   0
Time   360.82 Epot  -143.546 Temp(K)   47.92 Dener    885225.06 Salto   10 Error   0
Time   364.76 Epot  -144.878 Temp(K)   48.96 Dener    885225.06 Salto   10 Error   0
Time   369.83 Epot  -141.800 Temp(K)   49.43 Dener    885225.06 Salto   10 Error   0
Time   375.01 Epot  -141.632 Temp(K)   49.93 Dener    902239.94 Salto   11 Error   0
Time   380.61 Epot  -141.008 Temp(K)   49.98 Dener    902239.94 Salto   11 Error   0
Time   385.48 Epot  -143.427 Temp(K)   50.15 Dener    902239.94 Salto   11 Error   0
Time   389.51 Epot  -143.467 Temp(K)   50.04 Dener    902239.94 Salto   11 Error   0
Time   394.70 Epot  -145.386 Temp(K)   47.22 Dener    902239.94 Salto   11 Error   0
Time   398.94 Epot  -149.382 Temp(K)   50.70 Dener    902239.94 Salto   11 Error   0
Time   403.39 Epot  -146.749 Temp(K)   47.78 Dener    902239.94 Salto   11 Error   0
Time   407.54 Epot  -143.383 Temp(K)   46.82 Dener    902239.94 Salto   11 Error   0
Time   413.13 Epot  -145.554 Temp(K)   45.18 Dener    913511.81 Salto   12 Error   0
Time   417.16 Epot  -145.270 Temp(K)   48.06 Dener    913511.81 Salto   12 Error   0
Time   421.10 Epot  -144.460 Temp(K)   44.99 Dener    913511.81 Salto   12 Error   0
Time   425.32 Epot  -148.322 Temp(K)   51.24 Dener    913511.81 Salto   12 Error   0
Time   429.05 Epot  -146.800 Temp(K)   48.34 Dener    913511.81 Salto   12 Error   0
Time   433.08 Epot  -145.095 Temp(K)   43.92 Dener    913511.81 Salto   12 Error   0
Time   437.24 Epot  -143.363 Temp(K)   45.31 Dener    924004.19 Salto   13 Error   0
